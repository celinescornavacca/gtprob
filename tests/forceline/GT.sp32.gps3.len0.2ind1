(((((GA1,GA2),GA0),((HA1,HA2),HA0)),(((RA1,RA2),RA0),((VA0,(VA1,VA2)),((WA1,WA2),WA0)))),(((((((ZA1,ZA2),ZA0),((AB1,AB2),AB0)),((SA0,(SA1,SA2)),((((BB1,BB2),BB0),(((EB1,EB2),EB0),(FB0,(FB1,FB2)))),(((XA1,XA2),XA0),(YA0,(YA1,YA2)))))),(LA0,(LA1,LA2))),(BA0,(BA1,BA2))),(((((CB0,(CB1,CB2)),(DB0,(DB1,DB2))),((NA0,(NA1,NA2)),(OA0,(OA1,OA2)))),((AA1,AA2),AA0)),(((((TA0,(TA1,TA2)),(UA0,(UA1,UA2))),(DA0,(DA1,DA2))),((IA0,(IA1,IA2)),((MA0,(MA1,MA2)),(((PA1,PA2),PA0),((QA1,QA2),QA0))))),(((CA1,CA2),CA0),((((EA1,EA2),EA0),((FA1,FA2),FA0)),((JA0,(JA1,JA2)),(KA0,(KA1,KA2)))))))));
