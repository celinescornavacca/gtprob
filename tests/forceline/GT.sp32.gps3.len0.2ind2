(((((NA1,NA2),NA0),(OA0,(OA1,OA2))),(AA0,(AA1,AA2))),(((((HA0,(HA1,HA2)),((((WA0,(WA1,WA2)),((XA1,XA2),XA0)),((CB0,(CB1,CB2)),((DB1,DB2),DB0))),((((AB1,AB2),AB0),((BB1,BB2),BB0)),((EB0,(EB1,EB2)),((FB1,FB2),FB0))))),(((CA1,CA2),CA0),(DA0,(DA1,DA2)))),((((((((JA1,JA2),JA0),(KA0,(KA1,KA2))),((((LA1,LA2),LA0),((MA1,MA2),MA0)),((PA0,(PA1,PA2)),((QA1,QA2),QA0)))),(((YA0,(YA1,YA2)),(ZA0,(ZA1,ZA2))),((TA0,(TA1,TA2)),((UA0,(UA1,UA2)),(VA0,(VA1,VA2)))))),(EA0,(EA1,EA2))),(((FA1,FA2),FA0),(GA0,(GA1,GA2)))),(BA0,(BA1,BA2)))),(((RA0,(RA1,RA2)),((SA1,SA2),SA0)),(IA0,(IA1,IA2)))));
