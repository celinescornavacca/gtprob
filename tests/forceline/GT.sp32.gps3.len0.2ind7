(((((((AB0,(AB1,AB2)),(BB0,(BB1,BB2))),((NA1,NA2),NA0)),((((HA1,HA2),HA0),((IA1,IA2),IA0)),(((RA0,(RA1,RA2)),(SA0,(SA1,SA2))),((GA1,GA2),GA0)))),((BA0,(BA1,BA2)),((CA1,CA2),CA0))),(((((CB1,CB2),CB0),(DB0,(DB1,DB2))),((((KA1,KA2),KA0),(LA0,(LA1,LA2))),(((YA0,(YA1,YA2)),((ZA1,ZA2),ZA0)),((QA1,QA2),QA0)))),((((JA0,(JA1,JA2)),(((EB1,EB2),EB0),(FB0,(FB1,FB2)))),((((UA1,UA2),UA0),((VA1,VA2),VA0)),(((DA1,DA2),DA0),(EA0,(EA1,EA2))))),((((FA1,FA2),FA0),(((OA1,OA2),OA0),((PA1,PA2),PA0))),(((((WA1,WA2),WA0),(XA0,(XA1,XA2))),(TA0,(TA1,TA2))),((MA1,MA2),MA0)))))),(AA0,(AA1,AA2)));
