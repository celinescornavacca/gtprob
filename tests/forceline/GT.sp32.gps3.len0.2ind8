((((((LA1,LA2),LA0),(MA0,(MA1,MA2))),(EA0,(EA1,EA2))),(((GA0,(GA1,GA2)),((((QA1,QA2),QA0),((ZA0,(ZA1,ZA2)),((CB0,(CB1,CB2)),(DB0,(DB1,DB2))))),(NA0,(NA1,NA2)))),(((OA0,(OA1,OA2)),(PA0,(PA1,PA2))),((((RA1,RA2),RA0),(SA0,(SA1,SA2))),((TA0,(TA1,TA2)),(UA0,(UA1,UA2))))))),(((((((AB1,AB2),AB0),(BB0,(BB1,BB2))),(((EB1,EB2),EB0),((FB1,FB2),FB0))),((CA1,CA2),CA0)),((((JA0,(JA1,JA2)),(KA0,(KA1,KA2))),((DA0,(DA1,DA2)),((HA0,(HA1,HA2)),(IA0,(IA1,IA2))))),(((((FA1,FA2),FA0),((XA0,(XA1,XA2)),(YA0,(YA1,YA2)))),(BA0,(BA1,BA2))),(AA0,(AA1,AA2))))),(((VA1,VA2),VA0),((WA1,WA2),WA0))));
