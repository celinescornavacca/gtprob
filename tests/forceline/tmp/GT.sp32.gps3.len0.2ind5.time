2.62user 0.17system 0:03.08elapsed 90%CPU (0avgtext+0avgdata 148592maxresident)k
16inputs+8outputs (0major+17235minor)pagefaults 0swaps
