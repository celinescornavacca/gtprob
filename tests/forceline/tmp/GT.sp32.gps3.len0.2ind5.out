-141.796848365
